Lib/test/lock_tests.py

```diff
@@ -128,6 +128,7 @@ def f():
         self.assertFalse(result[0])
         lock.release()
 
+    @unittest.skip("TODO: RUSTPYTHON hangs")
```

Comment:
```suggestion
    @unittest.skip("TODO: RUSTPYTHON occasionally hangs")
```

Probably the word `occasional` will be helpful to prevent removing skip again before fixing it correctly

Reply:
ah, @fanninpm already did it in different PR. how about just pulling it?

Reply:
@youknowone I'm okay with you and/or @DimitrisJim cherry-picking those commits.